scopes/ui-foundation/use-box/menu/menu.tsx

```diff
@@ -1,55 +1,42 @@
-import React, { useState } from 'react';
+import React, { useState, useMemo, ReactNode } from 'react';
 import { Icon } from '@teambit/evangelist.elements.icon';
-import { BitInfo } from '@teambit/ui-foundation.ui.use-box.bit-info';
-import { Registry } from './registry';
-import { Install } from './install';
-import { Import } from './import';
+import { Tab } from '@teambit/ui-foundation.ui.use-box.tab';
+
 import styles from './menu.module.scss';
-import { Tabs } from './tabs';
-import { Elements } from './elements';
+
+// TODO where should we place this type? its being created in component ui runtime as well
```

Comment:
The menu is the one actually using it, it makes sense it put it in the menu (and have the componentUi make a slot of it